libs/blocks/global-footer/global-footer.js

```diff
@@ -273,7 +292,10 @@ class Footer {
 
     CONFIG.socialPlatforms.forEach((platform, index) => {
       const link = socialBlock.querySelector(`a[href*="${platform}"]`);
-      if (!link) return;
+      if (!link) {
+        lanaLog({ message: `Missing link for platform: ${platform}`, tags: 'errorType=warn,module=global-footer' });
```

Comment:
Is every platform always mandatory @overmyheadandbody ? I wonder if this might pollute the logs in case a missing link is an expected state.